doc/topics/tutorials/minionfs.rst

```diff
@@ -136,4 +156,4 @@ Or we can use a more elegant and salty way to add an SSH key:
 
 
 
-.. [*] Yes, that was the actual key on my server, but the server is already destroyed.
\ No newline at end of file
+.. [*] Yes, that was the actual key on my server, but the server is already destroyed.
```

Comment:
Thats not from me, I think the original author of that comment didnt want to get any lip about showing his key / keyart for the demonstration.